rpcs3/Emu/RSX/RSXThread.cpp

```diff
@@ -500,6 +500,29 @@ namespace rsx
 
 	void thread::operator()()
 	{
+		m_rsx_thread = std::this_thread::get_id();
```

Comment:
I'm pretty sure that this part is broken.
A: The shader dialog looks bad from time to time.
B: Crashes when closing
C: This moves shader compilation before spu compilation. This seems unintended.

I recommend creating a seperate PR for this.

Reply:
Fine.